src/clients/UBAClient/UBAClientUtilities.ts

```diff
@@ -232,11 +242,150 @@ export function getMostRecentBundles(
  * This can be used to validate a deposit that is much older than spoke pool client's lookback.
  * @param flow
  */
-// async function getBundleStateForFlow(flow: UbaFlow): ModifiedUBAFlow {
-//   // 1. First try to fetch flow information from existing bundle state saved in client memory.
-//   // 2. If not available, then we need to load additional data from the RPC.
-//   throw new Error("Unimplemented");
-// }
+export async function getModifiedFlow(
```

Comment:
This function isn't used anywhere but I do think it can be used to solve the problem where we want to validate a fill/deposit with a refund/fill but would need to extend the event RPC lookback